apps/prairielearn/src/pages/instructorQuestions/instructorQuestions.ts

```diff
@@ -0,0 +1,84 @@
+import ERR from 'async-stacktrace';
+import { Router } from 'express';
+import error = require('@prairielearn/error');
+import { logger } from '@prairielearn/logger';
+import sqldb = require('@prairielearn/postgres');
+import { QuestionAddEditor } from '../../lib/editors';
+import fs = require('fs-extra');
+import async = require('async');
+import { QuestionsPage } from './instructorQuestions.html';
+import { getQuestions } from '../../models/questions';
+
+const router = Router();
+const sql = sqldb.loadSqlEquiv(__filename);
+
+router.get('/', function (req, res, next) {
+  let questions;
+  async.series(
+    [
+      (callback) => {
+        fs.access(res.locals.course.path, (err) => {
+          if (err) {
+            if (err.code === 'ENOENT') {
+              res.locals.needToSync = true;
+            } else {
+              return ERR(err, callback);
+            }
+          }
+          callback(null);
+        });
+      },
```

Comment:
I'll give you bonus points if you rewrite this to use `async`/`await`, which should in turn mean we can get rid of `async.series` entirely here 🎉

Reply:
I'm going to pass on messing with this for now because I have lots of other things to do, both in this PR and beyond.